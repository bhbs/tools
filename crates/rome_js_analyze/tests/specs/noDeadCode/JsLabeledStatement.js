function JsLabeledStatement1() {
    label: while (true) {
        while (true) {
            if (true) {
                continue label;
            } else {
                break label;
            }
        }

        afterLabelJump();
    }
}
