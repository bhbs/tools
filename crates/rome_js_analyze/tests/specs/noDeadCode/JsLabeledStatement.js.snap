---
source: crates/rome_js_analyze/tests/spec_tests.rs
expression: JsLabeledStatement.js
---
# Input
```js
function JsLabeledStatement1() {
    label: while (true) {
        while (true) {
            if (true) {
                continue label;
            } else {
                break label;
            }
        }

        afterLabelJump();
    }
}

```


